client/src/modules/TeamDistribution/components/TeamDistributionCard/TeamDistributionCard.test.tsx

```diff
@@ -45,7 +51,14 @@ describe('TeamDistributionCard', () => {
 
   it('should render the edit and delete buttons for managers', () => {
     render(
-      <TeamDistributionCard distribution={distribution} isManager={true} onDelete={mockOnDelete} onEdit={mockOnEdit} />,
+      <TeamDistributionCard
+        distribution={distribution}
+        isManager={true}
+        onDelete={mockOnDelete}
+        onEdit={mockOnEdit}
+        onDeleteRegister={onDeleteRegister}
+        onRegister={onRegister}
+      />,
```

Comment:
You can extract the entire render(<TeamDistrubutionCard ... />) into separate function to change props in one place if needed. Check example client\src\modules\AutoTest\components\Coding\Coding.test.tsx